src/loadable-components.js

```diff
@@ -3,12 +3,21 @@ import loadable from "@loadable/component";
 import createLoadableVisibilityComponent from "./createLoadableVisibilityComponent";
 import { IntersectionObserver } from "./capacities";
 
-function loadableVisiblity(load, opts = {}) {
+function loadableVisiblity(
+  load,
+  opts = {},
+  intersectionObserverOptions = {
+    root: null,
+    rootMargin: "0px 0px 0px 0px",
```

Comment:
Shouldn't you have a default value as the eager boundaries we decided upon? And if someone wants to make it more lazy you do it?

Default - 0px 0px 0px 0px:
- passing down our eager value every single time in every import would be a much bigger hassle

Default - 0px 0px **1000px** 0px: 
- could there be edge cases where this could make things worse?